src/rollup.js

```diff
@@ -0,0 +1,118 @@
+import chalk from 'chalk';
+import { babelConfig } from './lib/babel';
+import { res, tryRequire } from './lib/util';
+
+/**
+ * @param {import('./configure').Options} options
+ */
+function getDefaultConfig(options) {
+	let babel = require('@rollup/plugin-babel').default;
+	let commonjs = require('@rollup/plugin-commonjs');
+	let nodeResolve = require('@rollup/plugin-node-resolve').default;
+
+	return {
+		output: {
+			format: 'iife',
+			name: `KarmaticTests`,
+			sourcemap: 'inline',
+		},
+		plugins: [
+			commonjs(),
+			nodeResolve(),
+			babel({
+				babelHelpers: 'bundled',
+				...babelConfig(options),
+			}),
+		].filter(Boolean),
+	};
+}
+
+/**
+ * @param {Object} pkg
+ * @param {import('./configure').Options} options
+ */
+function getRollupConfig(pkg, options) {
+	const ROLLUP_CONFIGS = ['rollup.config.js'];
+
+	let rollupConfig = options.rollupConfig;
+
+	if (pkg.scripts) {
+		for (let i in pkg.scripts) {
+			let script = pkg.scripts[i];
+			if (/\brollup\b[^&|]*(-c|--config)\b/.test(script)) {
+				let matches = script.match(
+					/(?:-c|--config)\s+(?:([^\s])|(["'])(.*?)\2)/
+				);
+				let configFile = matches && (matches[1] || matches[2]);
+				if (configFile) ROLLUP_CONFIGS.push(configFile);
+			}
+		}
+	}
+
+	if (!rollupConfig) {
+		for (let i = ROLLUP_CONFIGS.length; i--; ) {
+			// TODO: Breaks if rollup.config.js uses ES Modules... :(
+			rollupConfig = tryRequire(res(ROLLUP_CONFIGS[i]));
+			if (rollupConfig) break;
+		}
+	}
+
+	if (typeof rollupConfig === 'function') {
+		rollupConfig = rollupConfig({ karmatic: true });
+	}
+
+	if (Array.isArray(rollupConfig)) {
+		rollupConfig = rollupConfig[0];
+	} else if (rollupConfig && rollupConfig.then) {
+		rollupConfig = null;
+		console.warn(
+			chalk.yellow(
+				`Karmatic does not currently support asynchronous rollup configs. Using a default config instead.`
+			)
+		);
+	}
+
+	return rollupConfig || getDefaultConfig(options);
```

Comment:
Note to self: Consider adding code coverage babel plugin here if using existing config

Reply:
Done